static/js/vue-cdr-access/src/components/facets.vue

```diff
@@ -71,6 +72,17 @@
                     });
                 });
                 return display_list;
+            },
+
+            sortedFacetsList() {
+                return this.facetList.map((facet) => {
+                    if (facet.name === 'PARENT_COLLECTION') {
```

Comment:
I'd suggest changing this so it only sorts for the `CONTENT_TYPE` facet instead, since it is the only one with nested child values at present, and we wouldn't want to change the sort of some of the other potential facets.